site/_posts/2017-03-12-marching-ahead-embedded-ansible.md

```diff
@@ -0,0 +1,63 @@
+---
+title: "Last Week in ManageIQ: Marching ahead with the embedded Ansible integration
+author: lgalis
+date: 2017-03-12 21:33:30 -04:00
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+[Laura](https://github.com/lgalis) here bringing you this week's edition of Last Week in ManageIQ!
+
+## Featured
+We continue to make great progress as we move closer to the development complete milestone for our Fine release. This week, 
```

Comment:
The urls in the `Featured` section are not resolving correctly - I believe swapping out the `()` for `[]` will correct the problem.